iset.mm

```diff
@@ -106549,23 +106546,21 @@ which defines the set we are taking the supremum of must (a) be true at
       HWAXCRZXIXHXLXMVAAKVCVFUONQUQNVGOVHVIVJVKVL $.
   $}
 
-  ${
-    $( Extend ~ gcdmultiple so ` N ` can be an integer.  (Contributed by Scott
-       Fenton, 18-Apr-2014.)  (Revised by Mario Carneiro, 19-Apr-2014.) $)
-    gcdmultiplez $p |- ( ( M e. NN /\ N e. ZZ ) ->
+  $( Extend ~ gcdmultiple so ` N ` can be an integer.  (Contributed by Scott
+     Fenton, 18-Apr-2014.)  (Revised by Mario Carneiro, 19-Apr-2014.) $)
+  gcdmultiplez $p |- ( ( M e. NN /\ N e. ZZ ) ->
```

Comment:
If someone is going to do this please do it in both iset.mm and set.mm.

Reply:
Since this PR has been merged, I can sneak that into the next one.  @jkingdon it has already been done in set.mm.  I must have overlooked doing the same for iset.mm.  Sorry about that.